src/prefect/tasks/github/comments.py

```diff
@@ -0,0 +1,86 @@
+import json
+from typing import Any
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class CreateIssueComment(Task):
+    """
+    Task for creating a comment on the specified GitHub issue using the v3 version of the
+    GitHub REST API.
+
+    Args:
+        - repo (str, optional): the name of the repository to create the comment in; must be
+            provided in the form `organization/repo_name`; can also be provided to the `run`
+            method
+        - issue_number (int, optional): the ID of the issue to create the comment on.
+        - body (str, optional): the body of the comment; can also be provided to the `run` method
+        - **kwargs (Any, optional): additional keyword arguments to pass to the standard Task
+            init method
+
+    References:
+        - https://developer.github.com/v3/issues/comments/#create-an-issue-comment
```

Comment:
What do you think about adding a link like this to clarify which API end point each taks uses?

Reply:
I think [create_line](https://github.com/PrefectHQ/prefect/blob/f554d1313638286d4183b136f97b56bfbfae58db/docs/generate_docs.py#L129) needs to be updated to render `References` block properly

https://github.com/PrefectHQ/prefect/blob/f554d1313638286d4183b136f97b56bfbfae58db/docs/generate_docs.py#L129-L136

Reply:
I like having a `References` section, if you'd want to make the required changes here to support this I'd be +1 on it. I think your suggestion above should be sufficient. Would you be up to adding links like this to the other github tasks as well?

Reply:
@jcrist Gotcha. I'll make the change.